web/beacon-app/src/features/projects/types/projectQueryService.ts

```diff
@@ -14,3 +14,21 @@ export interface ProjectQueryDTO {
   projectID: string;
   query: string;
 }
+
+export interface MetaDataResponse {
+  key: string;
+  value: string;
+}
+export interface ProjectQueryResult {
+  metadata: MetaDataResponse[];
+  mimetype: string;
+  created: string;
+  is_base64_encoded: boolean;
+  data: any;
+  version: string;
+}
+
+export interface ProjectQueryResponse {
```

Comment:
The response could also return an error. https://github.com/rotationalio/ensign/blob/3dbfec37f617ca0e43a887becec06a0d890d36d5/pkg/tenant/api/v1/api.go#L289

Reply:
During the implementation of error handling, this aspect will likely be included. Currently, my primary focus is on displaying the results.